dashboards/solver-rewards-accounting/solver-totals.sql

```diff
@@ -26,13 +26,39 @@ per_solver_results as (
         batches_settled,
         num_trades
     from solver_data sd
+),
+
+jit_orders as (
```

Comment:
I think its always better to avoid acronoyms. I would also rather prefer liquidity_orders, instead of just-in_time orders.